### ISSUE TITLE
Removal of Zombie Pills

### ISSUE THREAD

**Issue by AdeptCL** [InZidiuZ/op-framework-issue-tracker#1966]
**Assurance:**
Yes ma'am.

**Summary:**
Removal of zombie pills, from any loot pool, **OR** make it an event (Christmas/Halloween/etc) based item.

**Reason:**
There is little to no roleplay from these zombie pills. There is no structure of finding contacts, building relationships, and obtaining guns. A new player with less than an hour in the server can run garbage, find a zombie pill, run it, and get a shotgun/pistol (mid to high tier based).

This completely destroys any form of roleplay when it comes to most played side, criminals. What is happening currently, is new players are finding these guns and selling them within 1 hour of roleplay, more often than groups that have been involved in the server for more than a few months that have developed serious storylines with gun suppliers, and having conflict with other groups over certain markets of guns.

My overall suggestion would be to remove the zombie pills from the regular loot pools, but the items that can be found within them, should be spread across more activities; Chop Shop, House Robberies, potentially even Diving. With the more normalised items (body armour, cuffs, etc) can be able to be found in Garbage, Transporting, etc.

I believe this will stabilise a lot of the worries people have currently, very much like the weed situation. People were forced to start fresh with it. It was known to be far too accessible, and people were avoiding roleplay. This is what is happening with the current use of zombie pills.

**Comment by LikeConnorr** [comment]
Should become a seasonal thing, maybe every halloween since it fits the theme? 
+ maybe a new item that can be implemented into the calendar for Christmas, but not as OP + doesn't stick around.

**Comment by xarxesthegreat** [comment]
they ruin a lot of RP and have cause more issues than theyre worth

**Comment by TalosTG** [comment]
Agreed. Zombie pills are like very random weapon caches that encourage a grindset mentality over a roleplay mentality.

**Comment by lildiavolo** [comment]
I remember when I first came into the city me and the boys, one of the biggest things on our mind was trying to get contacts to get something better than an ammunation pistol. Luckily, we knew Zane Mclane and he introduced us to Fonzo and that's how we got our first 50cal. and later on met up with AK for some PDWs and mid tier guns. 
Now adays, if you want a gun you just grind garbage and get a Z pill and pop it for some easy guns. 
Z pills just gotta go, it just brings no roleplay and if anything it takes away from it.
